### PR TITLE
Close button color test and start explore page test

### PR DESCRIPTION
**Pull request by broskoski**

### CODE REVIEW

File: apps/examples/examples.coffee [hunk 1]
```diff
@@ -3,158 +3,105 @@ module.exports = [
...
-    # channel_ids: [
-    #   "sublime",
-    #   "color-schemes",
-    #   "hotel-concepts",
-    #   "color-files",
-    #   "life-luxury",
-    # ]
-    channel_ids: [13771, 28613, 7469, 158542, 46371]
+    # channel_ids: ["year-zero", "color-files", "color-schemes", "hotel-concepts", "life-luxury"]
+    channel_ids: [37968, 28613, 7469, 158542, 46371]
```
Comment (dzucconi) [inline apps/examples/examples.coffee]: A simple thing but if you want to maintain a mapping between name and ID it'd probably alot clearer to use an object as a map:
```javascript
IDS = {
  "year-zero": 37968,
  "color-files": 28613,
  "color-schemes": 7469,
  "hotel-concepts": 158542,
  // ...
}

channel_ids: [
  IDS["year-zero"],
  IDS["color-files"],
  IDS["color-schemes"],
  IDS["hotel-concepts"],
  IDS["life-luxury"]
]
```

Comment (broskoski) [reply apps/examples/examples.coffee]: Ah yeah, good call.

File: react/components/FollowButton/index.js [hunk 2]
```diff
@@ -18,13 +20,16 @@ class FollowButton extends Component {
...
+    isLoggedIn: PropTypes.bool.isRequired,
   }
 
   toggleFollow = async () => {
     const {
-      id, type, data: { followable },
+      id, type, data: { followable }, isLoggedIn,
     } = this.props;
 
+    if (!isLoggedIn) { window.location = '/sign_up'; }
```
[the remark this answers is not in the document]
Comment (broskoski) [reply react/components/FollowButton/index.js]: I think it's ok to use it as a CTA (thinking of the /examples page in particular).

[the remark this answers is not in the document]
Comment (dzucconi) [reply react/components/FollowButton/index.js]: Yea

**Pull request merged**